sources/payment-hub/target/classes/log4j2.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    This Source Code Form is subject to the terms of the Mozilla
+    Public License, v. 2.0. If a copy of the MPL was not distributed
+    with this file, You can obtain one at
+
+    https://mozilla.org/MPL/2.0/.
+-->
+<Configuration status="INFO">
+    <Appenders>
```

Comment:
Shouldn't this rather be inside a resources classpath from where it gets picked instead of target/classes which should fall under .gitignore scope?